server/lib/models/account.go

```diff
@@ -22,7 +23,7 @@ func InitAccount() {
 	create_table(Account{}, "accounts", "Id")
 }
 
-func NewAccount(dbif Dbif, id *string, typ proto.Model_Account_Type, user string, mail string) (*Account, bool, error) {
+func NewAccount(dbif Dbif, id *proto.UUID, typ proto.Model_Account_Type, user string, mail string) (*Account, bool, error) {
```

Comment:
__[gometalinter]__
* [golint] exported function NewAccount should have comment or be unexported
* [gotype] undeclared name: proto